src/modules/layout/components/Header.js

```diff
@@ -130,14 +142,15 @@ class Header extends React.PureComponent<PropsType> {
   }
 
   render () {
-    const { navigation, t, theme } = this.props
+    const { cityModel, navigation, t, theme } = this.props
     const sharePath = navigation.getParam('sharePath')
+    const cityDisplayName = cityModel && `${cityModel.sortingName}${cityModel.prefix ? ` (${cityModel.prefix})` : ''}`
```

Comment:
That line is a little bit too complicated imo. 
Best way would be to create a component called `HeaderText` which accepts a cityModel as prop.
Easiest change would be to create a function which renders a HeaderText